js/web/lib/wasm/jsep/webgpu/gpu-data-manager.ts

```diff
@@ -252,13 +249,12 @@ class GpuDataManagerImpl implements GpuDataManager {
     gpuBufferForUploading.unmap();
 
     // GPU copy
-    const commandEncoder = this.backend.getCommandEncoder();
-    this.backend.endComputePass();
+    const commandEncoder = this.backend.device.createCommandEncoder();
```

Comment:
is it possible that there are existing pending commands in the current commandEncoder?

Reply:
I don't reuse the commandEncoder of backend, but create a new one for each stagingBuffer.